relnotes/taskext.feature.md

```diff
@@ -0,0 +1,64 @@
+## Built-in task and scheduler support in DaemonApp
+
+`ocean.util.app.DaemonApp` `ocean.util.app.ext.TaskExt`
+
+It is now possible to make `DaemonApp` initialize scheduler automatically using
+standard app configuration file. To do so, it is sufficient call
+`task_ext.enable` in the constructor:
+
+```D
+class MyApp : DaemonApp
+{
+    this ( )
+    {
+        super("name", "desc", VersionInfo.init);
+        this.task_ext.enable(); // NB: must be after `super` call
+    }
+```
+
+This extension is disabled by default to avoid clashing with existing
+application code which is already handling scheduler initialization in some way.
```

Comment:
Are daemonapps always going to use an scheduler? If so I guess the default should be changed in the next major? If so, maybe add an issue so we don't forget?

Reply:
Eventually, but AFAIK we have plenty of apps that haven't switched to task system yet. Also there is a problem with providing `PoolDescription` configuration which requires manual intervention anyway. I am not convinced there is any benefit in going for this direction.